assets/opengrep_rules/client/sources_gni.yaml

```diff
@@ -0,0 +1,27 @@
+rules:
+  - id: sources_gni
+    metadata:
+      author: Brian Johnson
+      references:
```

Comment:
https://github.com/brave/security-action/blob/main/assets/opengrep_rules/client/in-process-browser-test.yaml#L10C7-L10C28 we should add correctness or security in the metadata